modules/virt-importing-vm-datavolume.adoc

```diff
@@ -106,6 +106,7 @@ status: {}
 <1> The `HTTP` source of the image you want to import.
 <2> The `secretRef` parameter is optional.
 <3> The `certConfigMap` is required for communicating with servers that use self-signed certificates or certificates not signed by the system CA bundle. The referenced ConfigMap must be in the same namespace as the DataVolume.
+<4> Specify `type: dataVolume` or `type: ""`. If you specify any other value for `type`, such as `persistentVolumeClaim`, a warning is displayed and the virtual machine does not start.
```

Comment:
s/displayed/displayed,

Reply:
Fixed. 

I would have to verify with dev that "" defaults to dataVolume. I think it does, but need to verify --- I am not sure now. For me the primary thrust of this is not to use other values, esp. PVC, as that's where the user is getting into trouble. 

If we can merge it now, let's do so. Thanks for your help,

Bob

Reply:
The warning only shows if you try to use a PVC that is owned by a Datavolume instead of the Datavolume.

Reply:
When you confirm with dev, please open a follow-up PR to clarify that the options are interchangeable. I get that trying to fix the immediate issue is pressing, but it's worth adding clarity when you can.